service/src/main/java/com/derpgroup/quip/model/QuipStore.java

```diff
@@ -0,0 +1,107 @@
+package com.derpgroup.quip.model;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.derpgroup.quip.configuration.QuipConfig;
+import com.derpgroup.quip.manager.QuipRequestTypes;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class QuipStore {
+  private final Logger LOG = LoggerFactory.getLogger(QuipStore.class);
+  
+  private static QuipStore instance;
+  private boolean initialized = false;
+  
+  Map<QuipRequestTypes, String> sourceFiles = new HashMap<QuipRequestTypes,String>();
+  Map<QuipRequestTypes, Instant> lastUpdateTimes = new HashMap<QuipRequestTypes,Instant>();
+  Map<QuipRequestTypes, List<Quip>> quips = new HashMap<QuipRequestTypes,List<Quip>>();
```

Comment:
This should be storing a list of QuipGroups, not Quips.

Also, should it be a list? Or a LinkedHashMap, keyed off the groupName?